uaclient/entitlements/realtime.py

```diff
@@ -0,0 +1,53 @@
+from typing import Tuple
+
+from uaclient import messages, util
+from uaclient.entitlements import repo
+from uaclient.entitlements.base import IncompatibleService
+from uaclient.types import MessagingOperationsDict
+
+REALTIME_KERNEL_DOCS_URL = "https://ubuntu.com/realtime"  # TODO
```

Comment:
@nathan-j-hart @EdoardoBarbieriCanonical What is the URL that we can point users to for more information about the realtime kernel?

Reply:
We don't have one yet, but will work on it after the release

Reply:
Confirming here that the only URL we will have at launch will be the beta signup URL https://ubuntu.com/realtime-kernel. a docs page will come later

Reply:
Great that works! We just need something at that URL in time for releasing this

Reply:
:+1: The web team is on board, Scott is working on it currently.